### DISCUSSION TITLE
Three-phase fault at desired line position, some strange TDS results

### DISCUSSION THREAD

**Discussion by seedpower** [CURENT/andes#386] [Q&A] [answered]
Based on `ieee14_full.xlsx` , I create some cases.
I add a new virtual bus and a new line to implement fault at desired line position.

---
Case 1: Load factor 100%, fault occurs in line_12 at 80% positon during 1.0s~1.3s
[ieee14_1.00_Line_12_0.80_1.00_1.30.xlsx](https://github.com/cuihantao/andes/files/10046637/ieee14_1.00_Line_12_0.80_1.00_1.30.xlsx)

Below is TDS result:
```
-> Single process finished in 0.2790 seconds.
<Fault Fault_1>: Applying fault on Bus (idx=15) at t=1.0 sec.
<Fault Fault_1>: Clearing fault on Bus (idx=15) at t=1.3 sec.
100%|#######################################| 100.0/100 [00:00<00:00, 306.46%/s]
True
```

---
Case 2: Load factor 100%, fault occurs in line_12 at 90% positon during 1.0s~1.3s
[ieee14_1.00_Line_12_0.90_1.00_1.30.xlsx](https://github.com/cuihantao/andes/files/10046644/ieee14_1.00_Line_12_0.90_1.00_1.30.xlsx)

Below is TDS result:

```
-> Single process finished in 0.2761 seconds.
GENROU (vf range) out of typical upper limit.

   idx    | values | limit
----------+--------+------
 GENROU_1 | 7.000  | 5    
 GENROU_2 | 11.206 | 5    
 GENROU_3 | 5.413  | 5    
 GENROU_4 | 36.426 | 5    
 GENROU_5 | 12.359 | 5    


PQ.vcmp out of limits <vmin>

 idx   | Flag | Input Value | Limit
-------+------+-------------+------
 PQ_3  | zl   | 0.772       | 0.800
 PQ_4  | zl   | 0.782       | 0.800
 PQ_6  | zl   | -0.000      | 0.800
 PQ_7  | zl   | -0.000      | 0.800
 PQ_8  | zl   | -0.000      | 0.800
 PQ_11 | zl   | 0.369       | 0.800


ESST3A.HLI out of limits <VIMAX>

   idx    | Flag | Input Value | Limit
----------+------+-------------+------
 ESST3A_2 | zu   | 0.360       | 0.200
 ESST3A_3 | zu   | 0.278       | 0.200
 ESST3A_4 | zu   | 1.852       | 0.200
 ESST3A_5 | zu   | 0.633       | 0.200


Suspect initialization issue! Simulation may crash!

    Name      | Var. Value | Eqn. Mismatch
--------------+------------+--------------
 LL_y TGOV1 1 | 3.454      | -2.404       
 vil ESST3A 2 | 0.360      | -0.160       
 vil ESST3A 3 | 0.278      | -0.078       
 vil ESST3A 4 | 1.852      | -1.652       
 vil ESST3A 5 | 0.633      | -0.433       


Initialization failed!!
If you are developing a new model, check the initialization with
   andes -v 10 run -r tds --init C:\Users\Administrator\Desktop\andes\pycharm-test\ieee14_1.00_Line_12_0.90_1.00_1.30.xlsx
Otherwise, check the variables that are initialized out of limits.
  0%|                                                    | 0/100 [00:02<?, ?%/s]
Time step reduced to zero. Convergence is not likely.
Simulation terminated at t=0.0002 s.
```
---
Case 3: Load factor 100%, fault occurs in line_12 at 95% positon during 1.0s~1.3s
[ieee14_1.00_Line_12_0.95_1.00_1.30.xlsx](https://github.com/cuihantao/andes/files/10046662/ieee14_1.00_Line_12_0.95_1.00_1.30.xlsx)

Below is TDS result:
```
-> Single process exit with an error in 0.2863 seconds.
False
Power flow failed after 27 iterations for "C:\Users\Administrator\Desktop\andes\pycharm-test\ieee14_1.00_Line_12_0.95_1.00_1.30.xlsx".
Power flow not solved. Simulation will not continue.
```

Problem:
1. case2 can't continue TDS at t=0s ?
2. case3 power flow not sloved ?

**Comment by cuihantao** [comment]
Looks like your case 2 is not properly initialized. You will need to adjust the limits of some turbine governors and exciters accordingly.

For case 3, it's not due to the fault. Double check if any other parameters are modified or lines are accidentally removed.

**Comment by jinningwang** [comment]
Hi,

I will take a look at this issue and expect a response within one week.

Regards,
Jinning

**Comment by jinningwang** [comment, marked as answer]
Hi,

After investigation, the failure of Case2 and Case3 are caused by inappropriate parameters.

Case2 is caused by the limiter value, and Case3 is caused by the Bus initial voltage and angle.

More details can be found in the attached PDF.

Regards,
Jinning

[Discussion386.pdf](https://github.com/CURENT/andes/files/10073053/Discussion386.pdf)
